chips/imxrt1050/src/lib.rs

```diff
@@ -0,0 +1,271 @@
+//! Peripheral implementations for the IMXRT1050 MCU.
+//!
+//! imxrt1050 chip: <https://www.nxp.com/design/development-boards/i-mx-evaluation-and-development-boards/i-mx-rt1050-evaluation-kit:MIMXRT1050-EVK>
+
+#![crate_name = "imxrt1050"]
+#![crate_type = "rlib"]
+#![feature(llvm_asm, const_fn)]
+#![no_std]
+#![allow(unused_doc_comments)]
```

Comment:
Can you remove this allow and resolve the warnings it is hiding?

Reply:
Done.